src/olympia/files/tests/test_utils_.py

```diff
@@ -528,7 +528,7 @@ def test_resolve_i18n_message_locale_found():
 
 def test_resolve_i18n_message_uses_default_locale():
     messages = {
-        'en': {
+        'en-US': {
```

Comment:
This is how `messages` is being normalized already and passed in, the caller correctly assumes this.